docs/paper/admin/reference/system-properties.md

```diff
@@ -0,0 +1,178 @@
+---
+slug: /reference/system-properties
+---
+
+# Paper System Properties 
+
+These system properties can be set when you start your server allowing for the configuration of various settings.
+
+:::danger Danger Ahead
+
+Setting flags for the JVM can alter how it operates and the same goes for the Paper server.
+If you are unsure about what a flag does, it is recommended that you **do not use it**.
+
+:::
+
+## How they work
+
+System properties are set when you start your server. For example, if you are using a `.bat` or a `.sh` file to start your server, you can add the system properties to the file. For example:
+
+```bash
+java -Dpaper.log-level=FINE -jar paper.jar
+```
+
```

Comment:
I'd note here that windows (powershell?) requires quoting anything with `.`s in it. So something like `java "-Dpaper.log-level=FINE"`. Please double-check this, I'm unable to confirm at this moment.

Reply:
Checked it, You are right, have fixed